src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -10,7 +11,7 @@
 import java.util.List;
 
 @Repository
-public class PersonDAO {
+public class PersonDAO implements PersonDAOInterface {
```

Comment:
without interface, usually it is named like: PersonDAO and PersonDAOImpl/SimplePersonDAO for example